templates/catalog/book/bookInfo.tpl

```diff
@@ -1,100 +1,120 @@
-				<li><a href="#sharingTab">{translate key="submission.sharing"}</a></li>
-			{/if}
-		</ul>
-
-		<div id="abstractTab">
-			{$publishedMonograph->getLocalizedAbstract()|strip_unsafe_html}
-
-			{assign var=authors value=$publishedMonograph->getAuthors()}
-			{foreach from=$authors item=author}
-				{if $author->getIncludeInBrowse()}
-					<p>{translate key="catalog.aboutTheAuthor" roleName=$author->getLocalizedUserGroupName()}: <strong>{$author->getFullName()}</strong></p>
-					{assign var=biography value=$author->getLocalizedBiography()|strip_unsafe_html}
-					{if $biography != ''}{$biography}{else}{translate key="catalog.noBioInfo"}{/if}
-				{/if}
-			{/foreach}
-		</div>
-		{if $chapters|@count != 0}
-			<div id="contentsTab">
-				{foreach from=$chapters item=chapter}
-					<p>
-						<strong>{$chapter->getLocalizedTitle()}</strong>
-						{if $chapter->getLocalizedSubtitle() != '' }<br />{$chapter->getLocalizedSubtitle()}{/if}
-						{assign var=chapterAuthors value=$chapter->getAuthorNamesAsString()}
-						{if $publishedMonograph->getAuthorString() != $chapterAuthors}
-							<div class="authorName">{$chapterAuthors}</div>
-						{/if}
-					</p>
-				{/foreach}
-			</div>
-		{/if}
-		{if $availableFiles|@count != 0}
-		<div id="downloadTab">
-			{assign var=publicationFormats value=$publishedMonograph->getPublicationFormats()}
-			{assign var=currency value=$currentPress->getSetting('currency')}
-			{if !$loggedInUsername}<p>{translate key="catalog.loginRequiredForPayment"}</p>{/if}
-			{if $useCollapsedView}
-				<ul>
-					{foreach from=$publicationFormats item=publicationFormat}
-						{if $publicationFormat->getIsAvailable()}
-							{include file="catalog/book/bookFiles.tpl" availableFile=$availableFile publicationFormatId=$publicationFormat->getId() publishedMonograph=$publishedMonograph currency=$currency}
-						{/if}
-					{/foreach}
-				</ul>
-			{else}
-				{foreach from=$publicationFormats item=publicationFormat}
-					{assign var=publicationFormatId value=$publicationFormat->getId()}
-					{if $publicationFormat->getIsAvailable() && $availableFiles[$publicationFormatId]}
-						<div class="publicationFormatDownload" id="publicationFormat-download-{$publicationFormatId|escape}">
-							{$publicationFormat->getLocalizedName()|escape}
-							<ul>
-								{include file="catalog/book/bookFiles.tpl" availableFile=$availableFile publicationFormatId=$publicationFormatId publishedMonograph=$publishedMonograph currency=$currency}
-							</ul>
-						</div>
-					{/if}
-				{/foreach}
-			{/if}{* useCollapsedView *}
-		</div>
-		{/if}
-		{if !is_null($sharingCode) || !empty($blocks)}
-			<div id="sharingTab">
-				{$sharingCode}
-				{foreach from=$blocks item=block key=blockKey}
-					<div id="socialMediaBlock{$blockKey|escape}" class="pkp_helpers_clear">
-						{$block}
-					</div>
-				{/foreach}
-			</div>
-		{/if}
-	</div>
-</div>
+{**
+ * templates/catalog/book/bookInfo.tpl

```

Comment:
Did the line endings in this file accidentally get changed from UNIX-style to DOS-style? That would explain why the diff includes the whole file rather than just the changed portions. Please stick with UNIX-style line endings.